src/index.d.ts

```diff
@@ -312,7 +321,13 @@ interface Branch {
   ) => Promise<BranchUniversalObject>;
   handleATTAuthorizationStatus: (
     ATTAuthorizationStatus:ATTAuthorizationStatus
-  ) => void
+  ) => void;
+  getBranchQRCode: (
+    settings: BranchQRCodeSettings,
+    branchUniversalObject: BranchUniversalObject,
+    linkProperties: BranchLinkProperties,
+    controlParams: BranchLinkControlParams,
+  ) => Promise<string>;
```

Comment:
Will we not return an Image here?

Reply:
It seems like RN doesn't have something like UIImage from iOS or Image from Android. It seems to use either the URL, resource location, or the data URI, so I went with passing the base64 encoded string.

Reply:
Yeah going by https://reactnative.dev/docs/image that should be sufficient